app/src/main/java/org/grapheneos/apps/client/service/SeamlessUpdaterJob.kt

```diff
@@ -74,13 +74,13 @@ class SeamlessUpdaterJob : JobService() {
 
                 if (requireConfirmation.isNotBlank() && result.requireConfirmation.isNotEmpty()) {
                     if (content.isNotBlank()) content += ", "
-                    content += "$requireConfirmation update update available."
+                    content += "$requireConfirmation: update available!"
```

Comment:
I don't think we should have any of these exclamation marks in any of the notifications. I know there are some messages that have that but it should be removed.

Reply:
I'll remove some exclamation marks alongside then!